pages/lot/[id].js

```diff
@@ -240,7 +243,7 @@ export default function Lot({ id }) {
 }
 
 export async function getStaticPaths() {
-  const apiUrl = `https://data.cityoforlando.net/resource/5pzm-dn5w.json?application_type=Building Permit`;
+  const apiUrl = `https://data.cityoforlando.net/resource/5pzm-dn5w.json?application_type=Building Permit&worktype=New`;
```

Comment:
I realized that we only need the "new" building permits to get the list of lots

Reply:
Nicee